packages/fluentui/react-northstar/test/specs/utils/factories-test.tsx

```diff
@@ -2,15 +2,15 @@ import { callable } from '@fluentui/styles';
 import * as React from 'react';
 import * as _ from 'lodash';
 import { shallow } from 'enzyme';
-import { createShorthand, createShorthandFactory } from 'src/utils';
+import { createShorthandInternal as createShorthand, createShorthandFactory } from 'src/utils';
 import { Props, ShorthandValue, ObjectOf, ShorthandRenderFunction } from 'src/types';
 import { consoleUtil } from 'test/utils';
 
 // ----------------------------------------
 // Utils
 // ----------------------------------------
 
-type ShorthandConfig = {
+type GetShorthandConfig = {
```

Comment:
Is this change intentional?

Reply:
Yes, it conflicts with the actual ShorthandConfig, this however is used only in the test